packages/client/src/internals/TRPCClient.ts

```diff
@@ -82,7 +82,7 @@ export interface TRPCRequestOptions {
   /**
    * Pass additional context to links
    */
-  context?: OperationContext;
+  requestContext?: OperationContext;
```

Comment:
@KATT looks like we'll need to change our `OperationContext` name to avoid conflicts with Context Provider, wdyt?